File: IOT_Protocols/mqtt/publisher.js
const mqtt = require('mqtt')
const client = mqtt.connect('mqtt://localhost:1883');
const TOPIC = "diot";
const OPTIONS = {qos: 1};
client.on('connect', function ()
{
	console.log("Broker Connected!!");
	client.publish(TOPIC, '{temp: 24.5 C}', OPTIONS, function(err)
	{
		if(!err)
		{
			console.log("Published Success!");
		}
	})
})

File: IOT_Protocols/Paho-mqtt/publisher.py
import paho.mqtt.client as mqtt

print("Starting publisher...")

topic="diot"
QoS=0

client=mqtt.Client()

#client.on_connect=on_connect

def on_connect(client,userdata,flags,rc):
	if rc==0:
		print("connected to Broker")
		client.publish(top,"hello",QoS)

client.on_connect=on_connect
# Send Connect command/request to Broker
client.connect("localhost",1883)
